test/langtools/jdk/jshell/HangingRemoteAgent.java

```diff
@@ -35,9 +35,7 @@
  */
 class HangingRemoteAgent extends RemoteExecutionControl {
 
-    private static float timeoutFactor = Float.parseFloat(System.getProperty("test.timeout.factor", "1.0"));
-
-    private static final int TIMEOUT = (int)(2000 * timeoutFactor);
+    private static final int TIMEOUT = (int)(2000 * Double.parseDouble(System.getProperty("test.timeout.factor", "1.0")));
```

Comment:
why not `Utils.TIMEOUT_FACTOR`?

Reply:
There are a few places where I have changed java files that are not jtreg tests themself. The code is used by a jtreg test, but is not the "entry" into a test. Those files have no way to specify `@library` annotations, as no "test annotations" are parsed. It is a pity that a jtreg "library" can not specify dependencies to other "libraries".